frontend/src/Pagina/Inicio.jsx

```diff
@@ -1,16 +1,9 @@
-import { useNombres } from "../Hooks/obtenerNombres";
 import { useObtenerTonto } from "../Hooks/useObtenerTonto";
 import Tarjeta from "../Componentes/Tarjeta/Tarjeta";
-import Carga from "./Errores/Carga";
 
 const Inicio = () => {
-  const { cargaa } = useNombres();
   const { tonto } = useObtenerTonto();
```

Comment:
Hay algo raro acá. No aparecen mis cambios.

Fijate el PR que hice yo: https://github.com/The-Cowboys/the-cowboy-day/pull/51

Por qué no aparecen mis cambios acá???